backend/src/main/java/com/jobwait/stat/StatRefreshController.java

```diff
@@ -15,7 +15,7 @@ public class StatRefreshController {
     // stat id to timestamp
     private static Map<String, Instant> generateIdStampMap() {
         return Stats.known.stream().map(s -> s.id)
-                .collect(Collectors.toMap(Function.identity(), _ -> Instant.now(), (_, _) -> {
+                .collect(Collectors.toMap(Function.identity(), _ -> Instant.MIN, (_, _) -> {
```

Comment:
![image](https://github.com/user-attachments/assets/6f566463-538c-4cdc-a293-2976a78ffe80)
this guy does NOT read commit messages

(When I change it to min, the first request is guaranteed to surpass the lifetime window and be refetched, otherwise you'd have to wait)

Reply:
Ahh ok, my bad.